buildSrc/src/main/groovy/org/apache/beam/gradle/GrpcVendoring_1_36_0.groovy

```diff
@@ -49,7 +49,7 @@ class GrpcVendoring_1_36_0 {
       "io.grpc:grpc-netty:$grpc_version",
       "io.grpc:grpc-protobuf:$grpc_version",
       "io.grpc:grpc-stub:$grpc_version",
-      "io.netty:netty-transport-native-epoll:$netty_version",
+      "io.netty:netty-transport-native-epoll:$netty_version:linux-x86_64",
```

Comment:
Would you add comment why it's ok to limit this to linux-x86_64?

Or wasn't it working at all without classifier? https://netty.io/wiki/native-transports.html says

`<classifier>linux-x86_64</classifier>`

Reply:
You need to specify a classifier as the non-classifier only contains classes and no native code. Many other libs typically have a jar that contains all the native files necessary.

Note that I'm explicitly not adding support for aarch64.

Reply:
That makes sense.